<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThousandRectsInConcurentThreadsClass</class>
 <widget class="QWidget" name="ThousandRectsInConcurentThreadsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ThousandRectsInConcurentThreads</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="vert_splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="RenderWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QSplitter" name="horiz_splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: white
</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QCustomPlot" name="osc_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="settings_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>400</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </item>
      <item>
       <widget class="QFrame" name="check_box_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QCheckBox" name="chan_12">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>306</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>98</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_9">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>228</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>72</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_1">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>280</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_13">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>332</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>150</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_5">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>124</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_14">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>358</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_8">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>202</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>176</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_10">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>254</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="chan_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>46</y>
           <width>83</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderWidget</class>
   <extends>QWidget</extends>
   <header>twidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="thousandrectsinconcurentthreads.qrc"/>
 </resources>
 <connections/>
</ui>
